sources/osgAnimation/MorphGeometry.js

```diff
@@ -26,9 +26,10 @@ var MorphGeometry = function () {
     this._targetWeights = new Float32Array( 4 ); // Fixed length array feed by UpdateMorph
 
     this._morphTargetNames = undefined;
-    this._updateMorph = undefined;
 
     this._isInitialized = false;
+
+    this._maxMorphGPU = 4; // used by updateMorph to limit the number of morphed attributes done by the gpu
```

Comment:
the default value is repeated 3 times, maybe we could add MorphGeometry.MAX_MORPH_GPU (I dont remember the style for enum/constant)